Blockzilla/Utilities/URIFixup.swift

```diff
@@ -7,15 +7,17 @@ import Foundation
 class URIFixup {
     static func getURL(entry: String) -> URL? {
         let trimmed = entry.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
-        guard let escaped = trimmed.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlAllowed) else {
+        guard let escaped = trimmed.addingPercentEncoding(withAllowedCharacters: CharacterSet.alphanumerics) else {
```

Comment:
@OrlaM Would `urlAllowed` be more suitable?

Reply:
Double checking that this change is fine because I don't see the question answered..